repository/users.js

```diff
@@ -0,0 +1,44 @@
+import { User } from '../models';
+
+class UserRepository {
+  constructor(user) {
+    this._user = user;
```

Comment:
Here as well. At first i thought you are making a repo for a specific user but you are just holding here the `User` model.